vanilla-todo/src/observer.js

```diff
@@ -0,0 +1,45 @@
+const MILLISEC = 1000;
+const FRAMES = 60;
+const MILLI_PER_FRAME = MILLISEC / FRAMES;
+
+let currentObserver = null;
+
+const observe = (observer) => {
+  currentObserver = observer;
+  observer();
+  currentObserver = null;
+};
+
+const batch = (callback, limit = MILLI_PER_FRAME) => {
+  let timeout;
+  return () => {
+    clearTimeout(timeout);
+    timeout = setTimeout(() => {
+      callback();
+    }, limit);
+  };
```

Comment:
```suggestion
  let currentAnimationFrame;
  return () => {
    clearAnimationFrame(currentAnimationFrame);
    currentAnimationFrame = requestAnimationFrame(callback);
  };
```
이렇게 아예 `requestAnimationFrame`으로 해준다면 1frame 후에 실행 되도록 할 수 있답니다!